downloads/swan-lake-release-notes/swan-lake-2201.9.0/RELEASE_NOTE.md

```diff
@@ -617,6 +617,10 @@ public function main() returns error? {
 
 ### Improvements
 
+#### `http` package
+
+- Added TLS support for X25519Kyber768 key encapsulation.
```

Comment:
Shall we say something like this?
`Introduced X25519Kyber768 as a named group for key encapsulation in TLS.`

Reply:
```suggestion
- Introduced support for X25519Kyber768 as a named group for key encapsulation in TLS.
```